1002-no-detached-design-components.md

```diff
@@ -0,0 +1,18 @@
+---
+category: Design
+state: draft
+created: 2014-10-22
+updated: 2014-10-22
+---
+
+# Clarity design components 
+
+Design teams consuming Clarity Components in figma.
+
+## Guidance
+
+- Design teams **Must not** detach Clarity Components in figma.
```

Comment:
Design teams **Must Not** detach Clarity components in Figma. Doing so breaks alignment with the design system and makes it harder to maintain consistency.